runbotics/runbotics-desktop/src/core/bpm/runtime/runtime.service.ts

```diff
@@ -376,12 +376,14 @@ export class RuntimeService implements OnApplicationBootstrap, OnModuleDestroy {
         };
 
         const services = this.createEngineExecutionServices(processInstanceId);
+        const triggerData = request.triggerData;
```

Comment:
request?.triggerData -> in the interface you specified that triggerData is optional parameter